scripts/handlers/handlers.js

```diff
@@ -18,4 +19,9 @@ const toggleOpen = (event) => {
 
   they log any important information for developers
 
-*/
\ No newline at end of file
+*/
```

Comment:
Hi @razvanbrb, there is one thing I noticed after taking a second look to the tutorial, our listener is listening for transition events and fires the toggleActive function after the end of transition event. But there are more than one transition events and we need to trigger our function after the transition of flex property ends. So wes bos used ` if (e.propertyName.includes('flex'))` to determine if the transition even is happening on flex property. So maybe you can add that, but it may work without that line as well.